cloudfoundry-operations/src/main/java/org/cloudfoundry/operations/services/DefaultServices.java

```diff
@@ -184,6 +194,12 @@ private static ServiceInstanceType convertToInstanceType(String type) {
             .cast(AbstractServiceInstanceResource.class);
     }
 
+    private static Mono<AbstractUserProvidedServiceInstanceResource> createUserProvidedServiceInstance(CloudFoundryClient cloudFoundryClient, String spaceId,
+                                                                                                       CreateUserProvidedServiceInstanceRequest request) {
+        return requestCreateUserProvidedServiceInstance(cloudFoundryClient, spaceId, request.getName(), request.getCredentials(), request.getRouteServiceUrl(), request.getSyslogDrainUrl())
+            .cast(AbstractUserProvidedServiceInstanceResource.class);
```

Comment:
@antechrestos This can be simpler (no need for the `.cast()`) and inlined into the main flow.  I'll update.